src/PrestaShopBundle/Resources/views/Admin/Module/manage.html.twig

```diff
@@ -84,3 +84,9 @@
         </div>
     </div>
 {% endblock %}
+
+{% set js_translatable = {
+"Bulk Action - One module minimum": "You need to select at least one module to use the bulk action."|trans({}, 'Admin.Notifications.Success'),
+"Bulk Action - Request not found": "This bulk action request '[1]' does not exist."|trans({}, 'Admin.Notifications.Success'),
+"Bulk Action - Request not available for module": "Action [1] not available for module [2]. Skipped."|trans({}, 'Admin.Notifications.Success'),
```

Comment:
I'd say "The action "[1]" is not available ... "

Thank you, Thomas de Nabord!